OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs

```diff
@@ -109,7 +109,7 @@ public void ResolveOrder(Actor self, Order order)
 
 		public IEnumerable<SupportPowerInstance> GetPowersForActor(Actor a)
 		{
-			if (a.Owner != Self.Owner || !a.Info.HasTraitInfo<SupportPowerInfo>())
+			if (Powers.Count == 0 || a.Owner != Self.Owner || !a.Info.HasTraitInfo<SupportPowerInfo>())
```

Comment:
Is this still needed with the `Disabled` fix below? I guess it can't hurt...

Reply:
No it's not. It makes this even a bit more secure against crashes. (https://github.com/OpenRA/OpenRA/pull/17463#issuecomment-567207381 might have happened a different way.)